pkg/config/types/node.go

```diff
@@ -151,6 +151,7 @@ type NetworkConfig struct {
 	Type              string               `yaml:"Type"`
 	Port              int                  `yaml:"Port"`
 	AdvertisedAddress string               `yaml:"AdvertisedAddress"`
+	AuthSecret        string               `yaml:"AuthSecret"`
```

Comment:
@frrist – ok, that is now configurable here.

Reply:
<3